ros1_ign_bridge/src/convert_builtin_interfaces.cpp

```diff
@@ -904,24 +904,66 @@ template<>
 void
 convert_1_to_ign(
   const sensor_msgs::PointCloud2 & ros1_msg,
-  ignition::msgs::PointCloud & ign_msg)
+  ignition::msgs::PointCloudPacked & ign_msg)
 {
   convert_1_to_ign(ros1_msg.header, (*ign_msg.mutable_header()));
-
   std::cerr << "Unsupported conversion from [sensor_msgs::PointCloud2] to "
             << "[ignition::msgs::PointCloud]" << std::endl;
 }
 
 template<>
 void
 convert_ign_to_1(
```

Comment:
Since the messages are so similar, it shouldn't be a lot of trouble to add the reverse conversion, and some tests with some dummy data.